packages/neuron-wallet/src/env.ts

```diff
@@ -14,6 +15,7 @@ interface ENV {
 }
 const env: ENV = {
   isDevMode,
+  fileBasePath: path.resolve(app.getPath('userData'), isDevMode ? 'dev/' : ''),
```

Comment:
Check for test env as well?

Reply:
> Check for test env as well?

Yes, we might run e2e tests locally, this will avoid erasing production data.

Reply:
OK I didn't describe it clearly. I meant that we should have separate default folder for TEST environment from from development and production. For example:

* Production: ``
* Development: `dev/`
* Test: `test/`

This way test won't affect local development installation.

This might require change of `isDevMode` (to bring back `NODE_ENV` etc.).

Reply:
> OK I didn't describe it clearly. I meant that we should have separate default folder for TEST environment from from development and production. For example:
> 
> * Production: ``
> * Development: `dev/`
> * Test: `test/`
> 
> This way test won't affect local development installation.
> 
> This might require change of `isDevMode` (to bring back `NODE_ENV` etc.).

Agree, that's why I added basePath in the config, not in the code